### ISSUE TITLE
Feat: Add a config file to be able to ignore rules project-wide

### ISSUE THREAD

**Issue by gmuloc** [aristanetworks/j2lint#58]
cf: https://github.com/aristanetworks/j2lint/issues/53#issuecomment-1404697619

**Comment by wesley-dean** [comment]
Could one do something like this:

```bash
find -name '*.j2' -exec sh -c 'echo "$(cat j2lintrc) -- {}" | xargs -n99  j2lint ' \;
```

where `j2lintrc` would have lines like:

```
--flag
--flag2=\"foo\"
--flag3 \"foo bar bazzle\"
-f
```

which would result in:

```bash
j2lint --flag1 --flag2="foo" --flag3 "foo bar bazzle" -f -- ./test/bad.j2
j2lint --flag1 --flag2="foo" --flag3 "foo bar bazzle" -f -- ./test/good.j2
```

That is, as a work-around, could someone wrap `j2lint`? 

It wouldn't be a long-term solution, but at least it would give folks something they could do before such functionality is incorporated into the tool (if ever)

**Comment by qs5779** [comment]
If you do a web search for j2lint ignore files AI (Artificial Ignorance) says to create a .j2lintignore file in you project directory. That answer either is for another j2lint project or it is ignorant.

In my annoying case ansible-lint creates a .ansible directory in my project root and loads my required roles/collections and unfortunately several of those are not passing j2lint.

I like the idea of a .j2lintignore file in the project root or ~/.j2linitignore where I would put .ansible/ as the first entry.

My workaround in my justfile is:

jinja-lint: 
  #!/usr/bin/env bash
  set -euo pipefail
  echo "Running jinja lint ..."
  find . -name '*.j2' | grep -v -E '^./.ansible|^./globals' | xargs j2lint
